config/consensus.go

```diff
@@ -859,6 +863,10 @@ func initConsensusProtocols() {
 	// Can look up 2 assets to see asset parameters
 	v24.MaxAppTxnForeignAssets = 2
 
+	// Appears superfluous here in v24, but allows increasing the
+	// individual limits while maintaining max components in later vers.
+	v24.MaxAppTxnReferences = 8
```

Comment:
mb call it `MaxAppTxnTotalReferences` and comment is is set to v24.MaxAppTxnForeignAssets + v24.MaxAppTxnForeignApps + v24.MaxAppTxnAccounts.

Reply:
But in the future (vFuture!) it won't be the sum.  It's sum here, so that it has no effect on v24.  In vFuture, it's actually the same as the app or asa values.  That's to allow the teal code to freely decide which of the object types it needs more access to.  Maybe your point was to explain why it's 8 here.  I agree - will do.

And I do like your name better.  It's in keeping with how we added Total in some of the other combined PRs.